ads/caprofitx.js

```diff
@@ -14,14 +14,13 @@
  * limitations under the License.
  */
 
-import {loadScript, validateData} from '../3p/3p';
+import {loadScript} from '../3p/3p';
 
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function caprofitx(global, data) {
-  validateData(data, ['tagid'], []);
```

Comment:
Note: deleted because we will validate on our side.

Reply:
Sounds good.